### PR TITLE
docs - updated document loaders notebooks

### PR DESCRIPTION
**Pull request by leo-gan**
- Added links to the source sites with descriptions in consistent format
- Added code to install correspondent python packages (where it was needed)
- Fixed `example_data/facebook_chat.json` JSON (it was not the correct Json, which caused a parsing exception)

### CODE REVIEW

**Comment by skcoirz** [comment]
wow, that's a lot of changes. Thank you @leo-gan for the contribution! Just curious, are they auto generated or manually corrected? If auto, would really love to learn how to do it! Would get a lot of benefits from this new skill. :D Thank you so much!

**Comment by leo-gan** [comment]
> 

All manual, hehe

**Comment by skcoirz** [comment]
Passion and diligence. :D

File: docs/modules/indexes/document_loaders/examples/markdown.ipynb
Comment (vowelparrot) [inline docs/modules/indexes/document_loaders/examples/markdown.ipynb:1]: Oh interesting. Thanks for fixing the path

**Review by vowelparrot — COMMENTED** [review]
Looks great, @leo-gan ! I'll make a couple minor tweaks then land.

**Comment by vowelparrot** [comment]
This is fantastic, @leo-gan . Thank you for leading all these improvements!

File: docs/modules/indexes/document_loaders/examples/powerpoint.ipynb
[the remark this answers is not in the document]
Comment (leo-gan) [reply docs/modules/indexes/document_loaders/examples/powerpoint.ipynb:1]: Thanks!

**Pull request merged**